cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -90,17 +139,11 @@ scan_unifier_node::~scan_unifier_node(){}
  * input: -
  * output: -
  */
-void scan_unifier_node::getParams()
+void ScanUnifierNode::getParams()
 {
-
-  if(!pnh_.hasParam("loop_rate"))
-  {
-    ROS_WARN("No parameter loop_rate on parameter server. Using default value [100.0].");
-  }
-  pnh_.param("loop_rate", config_.loop_rate, (double)100.0);
-
   XmlRpc::XmlRpcValue topicList;
 
+  // TODO short parsing
```

Comment:
Link to list retrieval: http://wiki.ros.org/roscpp/Overview/Parameter%20Server#Retrieving_Lists